### ISSUE TITLE
Ignore node_modules folder

### ISSUE THREAD

**Issue by danilodeveloper** [insidersec/insider#21]
**Is your feature request related to a problem? Please describe.**
Hello guys!
Would be great Insider ignore the node_modules folder for Node.js projects. 

**Describe the solution you'd like**
Maybe create a CLI param like `-skip=node_modules`

**Describe alternatives you've considered**
I removed the node_modules folder before run Insider but it's not the best approach.

**Comment by hermescanutodesouza** [comment]
Thx !
Improving the idea I think about some thing like these 

```
-skipfolder=node_modules 
-skipfile=*.jpg|*.gif
```

**Comment by dvelardez** [comment]
I think you could exclude both files and folders in a single parameter.


Argument Name | Parameter | Description | Requirement
-- | -- | -- | --
--exclude | pattern | The path patterns to exclude from the scan - this option can be specified multiple times. This accepts Ant style path patterns (e.g. ‘directory/**/*.jar’). | Optional

**Comment by siewer** [comment]
does --exclude flag is included in 2.0.5 release? I get error:
```
gs:tmp gs$ ./insider -tech javascript -target /Users/gs/frontend -exclude /Users/gs/frontend/node_modules
flag provided but not defined: -exclude
```

**Comment by msAlcantara** [comment]
@siewer  We planning to implement the flag --exclude on next releases
